packages/react/src/tabs/indicator/TabsIndicator.tsx

```diff
@@ -44,9 +48,10 @@ const TabsIndicator = React.forwardRef<HTMLSpanElement, TabsIndicator.Props>(
       () => ({
         orientation,
         selectedTabPosition,
+        selectedTabSize,
```

Comment:
All fields from `state` are automatically translated into data attributes. This is not really needed (and incorrect in its current form, as it's serialized to `[object Object]`), so it must be explicitly excluded from style hooks (via `customStyleHookMapping` in line 67, similarly to `selectedTabPosition`).

Reply:
Fixed https://github.com/mui/base-ui/pull/1288/commits/8736ded65cf3b33c83be3b95ac9933887253106b